specification/microsoft.bing/data-plane/Common/Parameters.json

```diff
@@ -0,0 +1,57 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2017-08-30",
+    "title": "Common Referenced Parameters File",
+    "description": "File containing commonly referenced parameters."
+  },
+  "paths": {},
+  "parameters": {
+    "GlobalEndpoint": {
+      "name": "Endpoint",
+      "description": "Supported Microsoft.Bing endpoints (protocol and hostname, for example: \"https://westus.api.bing.microsoft.com\", \"https://api.bing.microsoft.com\").",
+      "x-ms-parameter-location": "client",
+      "required": true,
+      "type": "string",
+      "in": "path",
+      "x-ms-skip-url-encoding": true,
+      "default": "https://api.bing.microsoft.com"
+    },
+    "ImageStream": {
```

Comment:
Double check but I strongly suspect ImageStream and ImageUrl aren't needed for this service.